content/en/docs/how-tos/interact-with-running-jobs.md

```diff
@@ -286,3 +286,243 @@ $ openshift-tests run
+
+### Running `openshift-tests`
+
+Once the above requirements are met, run the `openshift-tests` binary for one of these scenarios:
+
+* Running the "openshift/conformance/serial" conformance suite
+
+  Using an aws cluster, here's an example command to run the openshift/conformance/serial tests:
+
+  ```bash
+  ~/openshift-tests run openshift/conformance/serial \
+         --provider '{"type":"aws", "region":"us-west-1", "zone":"us-west-1","multizone":true,"multimaster":true}' \
+         -o $mydir/logs/artifacts/e2e.log --junit-dir $mydir/logs/artifacts/junit
+  ```
+
+* Run the upgrade test:
+
+  Using an aws cluster, here's an example command to run an upgrade tests:
+
+  ```bash
+  ~/openshift-tests run-upgrade all --to-image 4.11.0-0.nightly-2022-06-15-161625 \
+         --options '' \
+         --provider '{"type":"aws", "region":"us-west-1", "zone":"us-west-1","multizone":true,"multimaster":true}' \
+         -o $mydir/logs/artifacts/e2e.log --junit-dir $mydir/logs/artifacts/junit
+  ```
+
+  Using an gcp cluster, here's an example command to run an upgrade tests:
+
+  ```bash
+  ~/openshift-tests run-upgrade all --to-image 4.11.0-0.nightly-2022-06-15-161625 \
+         --options '' \
+         --provider '{"type":"gce", "region":"us-west-1","multizone": true,"multimaster":true,"projectid":"<yourProjectID>"}' \
+         -o $mydir/logs/artifacts/e2e.log --junit-dir $mydir/logs/artifacts/junit
+  ```
+
+  NOTE: be careful to use `"type":"gce"` and not`"type":"gcp"`; the latter is wrong and will result in errors.
+
+When your test finishes, you will see familiar files (similar to a prow job) in the output directories:
+
+```bash
+$ tree tmp/logs
+ artifacts
+ ├── e2e.log
+ └── junit
+     ├── AdditionalEvents__sig_api_machinery_Kubernetes_APIs_remain_available_for_new_...
+     ├── AdditionalEvents__sig_api_machinery_Kubernetes_APIs_remain_available_with_reused_...
+	 ...
+     ├── AdditionalTest_cluster-upgrade_2022-06-17T08:53:51-04:00.json
+     ├── alerts_20220617-124636.json
+     ├── backend-disruption_20220617-124636.json
+     ├── e2e-events_20220617-124636.json
+     ├── e2e-timelines_e2e-namespaces_20220617-124636.html
+     ├── e2e-timelines_e2e-namespaces_20220617-124636.json
+	 ...
+     ├── junit_e2e_20220617-125422.xml
+     ├── junit_upgrade_1655470431.xml
+     ├── pod-placement-data.json
+     ├── pod-transitions.txt
+     ├── resource-events_20220617-124636.zip
+     └── resource-pods_20220617-124636.zip
+```
+
+### TIPs
+
+This section contains tips and notes to keep in mind when running the tests suites.
+
+* Before running your tests, ensure the `artifacts` and `junit` directories are empty; if you don't, you may have a hard time
+  distinguishing what output files belong to what invocation of `openshift-tests`.
+* An upgrade tends to take close to an hour.  To confirm an upgrade is happening and progressing, use these commands:
+
+  ```bash
+  $ export KUBECONFIG=cluster-bot...txt
+  $ oc get clusterversion version
```

Comment:
You could add -w for watch here, which would print a line every time the % changes I think.

Reply:
I'll try that sometime.